raddb/mods-config/sql/main/postgresql/queries.conf

```diff
@@ -27,6 +27,25 @@
 
 sql_user_name = "%{User-Name}"
 
+#######################################################################
+#  Query config:  Event-Timestamp
+#######################################################################
+# event_timestamp_epoch is the basis for the time inserted into
+# accounting records. Typically this will be the Event-Timestamp of the
+# accounting request, which is usually provided by a NAS.
+#
+# Uncomment the next line, if you want the timestamp to be based on the
+# request reception time recorded by this server, for example if you
+# distrust the provided Event-Timestamp.
+#event_timestamp_epoch = "%l"
+
+event_timestamp_epoch = "%{%{integer:Event-Timestamp}:-%l}"
+
+# event_timestamp is the SQL snippet for converting an epoch timestamp
+# to an SQL date.
+
+event_timestamp = "TO_TIMESTAMP($event_timestamp_epoch)"
```

Comment:
That should be:

event_timestamp = "TO_TIMESTAMP(${event_timestamp_epoch})"

You need the `{}` around all variable names in order to expand them.

Reply:
You just beat me to it. I didn't realise that your earlier comments referred to Travis failures which I've just noticed.

I'll push a fix.